### ISSUE TITLE
Intercom cookie without SameSite

### ISSUE THREAD

**Issue by wilzbach** [storyscript/studio#108]
A cookie associated with a cross-site resource at http://intercom.io/ was set without the `SameSite` attribute. A future release of Chrome will only deliver cookies with cross-site requests if they are set with `SameSite=None` and `Secure`. You can review cookies in developer tools under Application>Storage>Cookies and see more details at https://www.chromestatus.com/feature/5088147346030592 and https://www.chromestatus.com/feature/5633521622188032.

See also:
- https://www.chromium.org/updates/same-site
- https://community.intercom.com/t/cookie-warning-in-chrome/1397

tl;dr: needs to be fixed by the Intercom people, but opening it here as this will break the Intercom window soon and workarounds might need to be found.

**Comment by JeanBarriere** [comment]
We can't really do anything about that except waiting for Intercom to update their library
